llama-index-integrations/tools/llama-index-tools-passio-nutrition-ai/README.md

```diff
@@ -0,0 +1,25 @@
+# Passio Nutrition AI Tool
+
+This tool connects to a Passio Nutrition AI account and allows an Agent to perform searches against a database of over 2.2M foods.
+
+You will need to set up a search key using Passio Nutrition API,learn more here: https://www.passio.ai/nutrition-ai#nutrition-api-pricing
+
+## Usage
+
+Here's an example usage of the NutritionAIToolSpec.
```

Comment:
NIT: you may want to make mention here that `llama-index-agents-openai` needs to be installed in addition to this extension.